### ISSUE TITLE
Method to get weight

### ISSUE THREAD

**Issue by Uhouhzam** [DariusAf/MesoNet#31]
Hello, I am a student of master. I work for my research project deepfake detection. I wanted to test your code with other deepfake dataset. I used your code Meso4 and MesoInception to test other deepfake dataset. It doesn't work as better as writed in the papier. I think it's problem of weight. Can I know what kind of method that you used to get your weight, please.  Did you use backpropagation to get your weight and save them as file ?

**Comment by DariusAf** [comment]
Backpropagation would give you the gradient of the weight, not their value. It was trained in a very standard way that is described in the paper. And saved using `tf.keras.Model.save_weights()` to the H5 format, available on this repo.

And yes, the method was one of the first paper on deepfake detection, the scope was simply to train on a one of the first existing deepfake dataset. It was never claimed that it works out-of-distribution or out-of-task, though this is an important aspect that has been studied by subsequent papers.
